motorCmd[] setPoint
int32[] slaveAddress